jme3-core/src/main/java/com/jme3/anim/tween/action/Action.java

```diff
@@ -105,4 +192,29 @@ public void cloneFields(Cloner cloner, Object original) {
         actions = cloner.clone(actions);
         mask = cloner.clone(mask);
     }
+    
+    /**
+     * Tests the value of the forward flag.
+     *
+     * @return true if the animation action is running forward, false otherwise.
+     */
+    protected boolean isForward() {
+        return forward;
+    }
+
+    /**
+     * Adjusts the forward flag which controls the animation action directionality.
+     *
```

Comment:
Ditto here.

Reply:
This is a `protected` access method, by convention, it must have documentation in case someone else extended this class or its subclasses.